### PR TITLE
Add support for retrieving system service --timeout

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
Fixes: https://github.com/containers/podman/issues/11618

**Review by Luap99 — COMMENTED** [review]
LGTM

**Comment by TomSweeneyRedHat** [comment]
Any doc/man page changes?

File: cmd/podman/system/service.go [hunk 1]
```diff
@@ -52,8 +52,9 @@ func init() {
 
 	flags := srvCmd.Flags()
 
+	cfg := registry.PodmanConfig()
 	timeFlagName := "time"
-	flags.Int64VarP(&srvArgs.Timeout, timeFlagName, "t", 5, "Time until the service session expires in seconds.  Use 0 to disable the timeout")
```
Comment (mheon) [inline cmd/podman/system/service.go:56]: Does this actually work? This flag requires an argument, and it's not enabled unless explicitly specified. The default is always going to be overridden?

Comment (Luap99) [reply cmd/podman/system/service.go:56]: Yes that is how it works. The same logic is used for many flags in other commands.

Comment (mheon) [reply cmd/podman/system/service.go:56]: So - how does this help us, then? The unit file is going to have to specify `-t <value>` which overrides the default we're setting here, so `containers.conf` is getting ignored.

Comment (Luap99) [reply cmd/podman/system/service.go:56]: Which unit includes the `-t` flag?
```
$ systemctl cat podman.service 
# /usr/lib/systemd/system/podman.service
[Unit]
Description=Podman API Service
Requires=podman.socket
After=podman.socket
Documentation=man:podman-system-service(1)
StartLimitIntervalSec=0

[Service]
Type=exec
KillMode=process
Environment=LOGGING="--log-level=info"
ExecStart=/usr/bin/podman $LOGGING system service

[Install]
WantedBy=multi-user.target
```

Comment (Luap99) [reply cmd/podman/system/service.go:56]: https://github.com/containers/podman/blob/main/contrib/systemd/system/podman.service

Comment (mheon) [reply cmd/podman/system/service.go:56]: OH - having a timeout is the default, you have to explicitly opt out with `-t 0`.

OK, this works, then. Nevermind.

**Pull request merged**